lib/travis/api/v3/renderer.rb

```diff
@@ -31,6 +34,21 @@ def href(type, string_args = nil, script_name: nil, **args)
       expander.call(args)
     end
 
+    def render_model(model, type: model.class.name[/[^:]+$/].to_sym, mode: :minimal, **options)
+      Renderer[type].render(model, mode, **options)
+    end
+
+    def render_value(value, **options)
+      case value
+      when Hash        then value.map { |k, v| [k, render_value(v)] }.to_h
+      when Array       then value.map { |v   | render_value(v)      }
+      when *PRIMITIVE  then value
+      when Time        then value.strftime('%Y-%m-%dT%H:%M:%SZ')
+      when Model       then render_model(value, **options)
+      else raise ArgumentError, 'cannot render %p (%p)' % [value.class, value]
+      end
+    end
+
```

Comment:
This part just moved (so an error payload can include a minimal repo payload, see specs).